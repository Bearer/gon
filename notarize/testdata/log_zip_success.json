{
  "logFormatVersion": 1,
  "jobId": "3382aa04-e417-46a0-b1b4-42eebf85906c",
  "status": "Accepted",
  "statusSummary": "Ready for distribution",
  "statusCode": 0,
  "archiveFilename": "gon.zip",
  "uploadDate": "2019-11-06T00:51:10Z",
  "sha256": "1070be725b5b0c89b8dad699a9080a3bf5809fe68bfe8f84d6ff4a282d661fd1",
  "ticketContents": [
    {
      "path": "gon.zip/foo",
      "digestAlgorithm": "SHA-256",
      "cdhash": "b7049085e21423f102d6119bca93d57ebd903289",
      "arch": "x86_64"
    }
  ],
  "issues": null
}
