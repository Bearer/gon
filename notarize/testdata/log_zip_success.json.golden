(*notarize.Log)({
 JobId: (string) (len=36) "3382aa04-e417-46a0-b1b4-42eebf85906c",
 Status: (string) (len=8) "Accepted",
 StatusSummary: (string) (len=22) "Ready for distribution",
 StatusCode: (int) 0,
 ArchiveFilename: (string) (len=7) "gon.zip",
 UploadDate: (string) (len=20) "2019-11-06T00:51:10Z",
 SHA256: (string) (len=64) "1070be725b5b0c89b8dad699a9080a3bf5809fe68bfe8f84d6ff4a282d661fd1",
 Issues: ([]notarize.LogIssue) <nil>,
 TicketContents: ([]notarize.LogTicketContent) (len=1 cap=4) {
  (notarize.LogTicketContent) {
   Path: (string) (len=11) "gon.zip/foo",
   DigestAlgorithm: (string) (len=7) "SHA-256",
   CDHash: (string) (len=40) "b7049085e21423f102d6119bca93d57ebd903289",
   Arch: (string) (len=6) "x86_64"
  }
 }
})
